distribution/client/src/lib/error-telemetry.ts

```diff
@@ -5,13 +5,24 @@
 * handle both error and "metrics" telemetry, but let's start with a smaller
 * scope at least for now.
 */
-const { Tail } = require('tail');
-const fs       = require('fs');
-const logger   = require('winston');
-const path     = require('path');
-const Storage  = require('./storage');
+import { Tail } from 'tail';
+import fs from 'fs';
+import * as logger from 'winston'
+import path from 'path';
+
+import Storage from './storage';
+
+export interface ErrorTelemetryOptions {
+  flavor?: string,
```

Comment:
as I seem to understand, I _think_ this should actually be non optional, but that's not critical anyway and fixable later since the rest of the code makes it crash anyway.